docs/api/schema.yml

```diff
@@ -1814,15 +1832,16 @@ components:
         github_users:
           type: array
           items:
-            type: string
+            $ref: '#/components/schemas/GitHubUser'
         repo_image_url:
           type: string
           format: uri
           readOnly: true
         org_config_names:
           type: array
           items:
-            type: string
+            type: object
+            additionalProperties: {}
```

Comment:
Should/can we be more specific about the properties of these objects?

```ts
interface OrgConfig {
  key: string;
  label?: string;
  description?: string;
}
```

Reply:
We definitely can, just need to figure out a way to handle arbitrary keys. Or should we drop them when outputting via the API?

Reply:
I think it'd be fine to drop everything except `key`, `label`, and `description`.